aclk/aclk.c

```diff
@@ -320,6 +320,15 @@ static int handle_connection(mqtt_wss_client client)
             return 1;
         }
 
+        if (aclk_kill_link) { // User has reloaded the claiming state
```

Comment:
can we change it to :
`if (disconnect_req || aclk_kill_link) {`
to not duplicate code?

Reply:
Yes @underhood , you're right. Will change it. Had it on different block for some checks, it doesn't need to be in it's own block.